apps/api/src/app/environments-v1/usecases/create-environment/create-environment.usecase.ts

```diff
@@ -67,6 +97,21 @@ export class CreateEnvironment {
       });
     }
 
+    await this.createNovuIntegrationsUsecase.execute(
+      CreateNovuIntegrationsCommand.create({
+        environmentId: environment._id,
+        organizationId: environment._organizationId,
+        userId: command.userId,
+      })
+    );
+
     return environment;
   }
+
+  private getEnvironmentColor(name: string, commandColor?: string): string | undefined {
+    if (name === EnvironmentEnum.DEVELOPMENT) return '#ff8547';
+    if (name === EnvironmentEnum.PRODUCTION) return '#7e52f4';
+
+    return commandColor;
```

Comment:
The use case is missing strict color validation due to its reusability manner, so we need to make sure we validate it before creating a new env.

```suggestion
	if(!commandColor){
	    throw Error....
	}

    return commandColor;
```

Reply:
I aggree, we have it on the controller DTO, but will add it here too